frontend/hub/HubMasthead.tsx

```diff
@@ -22,7 +22,7 @@ export function HubMasthead() {
   const openAnsibleAboutModal = useAnsibleAboutModal();
   const { clearAllCache } = useClearCache();
   const navigate = useNavigate();
-  const activeUser = useActiveUser();
+  const activeUser = useAwxActiveUser();
```

Comment:
This one needs to come from the hubContext. See: https://github.com/ansible/ansible-ui/pull/1375